ui/src/fixturemanager.cpp

```diff
@@ -504,11 +630,8 @@ void FixtureManager::fixtureSelected(quint32 id)
         createInfo();
 
     m_info->setText(QString("%1<BODY>%2</BODY></HTML>")
-                    .arg(fixtureInfoStyleSheetHeader())
-                    .arg(fxi->status()));
-
-    // Enable/disable actions
-    slotModeChanged(m_doc->mode());
```

Comment:
Why did you delete this line? In your opinion it is not necessary?

Reply:
This isn't needed IMO indeed: `fixtureSelected()` is only ever called from `slotSelectionChanged()` and `slotSelectionChanged()` has an explicit `slotModeChanged(m_doc->mode());` at the end of that function. When inspecting the runtime: `slotModeChanged()` is called twice every time you select a single fixture. When you select a group for instance, this is not the case (`fixtureGroupSelected()` does not have `slotModeChanged(m_doc->mode());` at the end).

I removed it to avoid running that call twice and to keep things consistent with the other flows, unless I'm missing something?